### ISSUE TITLE
Training options of the example problems

### ISSUE THREAD

**Issue by huweiATgithub** [Tenavi/HJB_NN#3]
Hi Tenavi, 

Thank you for sharing your code. Would you share your detailed training options for the example problems, satellite and burgers?
In your code, it is
```
self.training_opts = {
            'weight_A': [10.],
            'BFGS_opts': {}
            }
```

Thanks,
Wei

**Comment by Tenavi** [comment]
Hi Wei,

You're right this is not the most transparent way of passing these options. It's leftover from when I was first writing the code.

I changed the code now to set options in the config file in a more intuitive way. Hopefully this makes more sense and feel free to email me with any questions.

Best,
Tenavi

**Comment by huweiATgithub** [comment]
Thank you for your reply! I tried sending an email to you but failed. 

I see that you added default options to the problem_def.py file, which was default options defined in the train function.
I have two questions:
•	The candidate points to pick when adaptive sampling data is set to 2, why not set to a larger number? I mean, it is very cheap to compute gradient for many points using partially trained neural network;
•	The maximal rounds is set to 1, then the for loop in train is supposed to run just for one time and convergence test is not performed. Is that enough for the network to converge?

Thank you,
Wei

**Comment by Tenavi** [comment]
1. The number of candidate samples is definitely something to experiment with. I found that setting it too high made training less accurate eventually. I think this feature may be more useful when the value function is more complicated.
2. That's correct, no convergence test will be performed with the maximum round set to 1. I think it's best to start with 1 round to get a feel for how different hyperparameters affect training before changing this.

**Comment by huweiATgithub** [comment]
OK, thank you.
